src/test/java/uk/gov/hmcts/reform/et/model/service/hearingvalues/ServiceHearingValuesTest.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.et.model.service.hearingvalues;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.et.utils.ResourceLoader;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+class ServiceHearingValuesTest {
+
+    private static final ObjectMapper MAPPER = new ObjectMapper().registerModule(new JavaTimeModule());
+
+    @Test
+    void shouldSerializeToPoJoSuccessfully() throws Exception {
+
+        String actualJson = ResourceLoader.loadJson("serviceHearingValues.json");
```

Comment:
As discussed previously, is there a valid reason for why we need to load json files for testing here?
Can we not use Lombok builder? just conscious that if we go down this path it will become messy soon like some other repos.

Reply:
I have removed the json file and used builder to build the class instead.